install.sh

```diff
@@ -110,6 +120,21 @@ prepare_user() {
   return 0
 }
 
+run_as_user() {
+  sudo -u $USER /bin/bash -c "
```

Comment:
> But `-H env PATH=...` doesn't work because on CentOS 7, by default,

Really? It seems work well on http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/o
pscode_centos-7.1_chef-provisionerless.box Vagrant box.

```text
% sudo -u root -H env PATH=/bin: /bin/env | grep PATH
PATH=/bin:
SUDO_COMMAND=/bin/env PATH=/bin: /bin/env
```